godotproject/Level0.tscn

```diff
@@ -23,7 +23,6 @@ Star = ExtResource( 9 )
 
 [node name="Player" parent="." instance=ExtResource( 1 )]
 position = Vector2( 110.163, 870.544 )
-scale = Vector2( 0.6, 0.6 )
```

Comment:
The frog's scale had to be changed from 0.6 to 1 because it was messing up the tongue display. The frog looks the right size on screen now imo. It doesn't seem that any of the movement was affected by this, other than the frog tongue, which had its values adjusted to compensate.